lti_consumer/tests/unit/test_lti_xblock.py

```diff
@@ -1623,6 +1676,9 @@ def test_get_lti_1p3_launch_data(self, pii_sharing_enabled, ask_to_send_username
         # Mock out get_pii_sharing_enabled to reduce the amount of mocking we have to do.
         self.xblock.get_pii_sharing_enabled = Mock(return_value=pii_sharing_enabled)
 
+        # Mock custom_parameters property.
+        self.xblock.get_lti_1p3_custom_parameters = Mock(return_value={})
```

Comment:
Could you check this `self.xblock.get_lti_1p3_custom_parameters.assert_called_once_with()` ?. @kuipumu